server/bin/gold/test-5.2.txt

```diff
@@ -72,15 +50,15 @@ Index:  pbench-unittests.server-reports.1900-01 1
 len(actions) = 1
 [
     {
-        "_id": "6cf3f985bead14580feb9096107ff1ba",
+        "_id": "9eed33308461c90c11fbc719ebbd9c42",
         "_index": "pbench-unittests.server-reports.1900-01",
         "_op_type": "create",
         "_source": {
             "@timestamp": "1900-01-01T00:00:00",
             "chunk_id": 1,
             "doctype": "status",
             "name": "pbench-unpack-tarballs",
-            "text": "pbench-unpack-tarballs.run-1900-01-01T00:00:00-UTC(unit-test) - w/ 0 errors\nProcessed 7 result tar balls, 7 successfully, 0 warnings, 0 errors, and 0 duplicates\n\n",
+            "text": "pbench-unpack-tarballs.run-1900-01-01T00:00:00-UTC(unit-test) - w/ 0 errors\nProcessed 4 result tar balls, 4 successfully, 0 warnings, 0 errors, and 0 duplicates\n\n",
```

Comment:
We need to fix up the starting state of `test-5.2` so that the tar balls that are currently in an `001` reception area are moved to `002` now.

Reply:
The starting state that @portante mentions is stored in `state/test-5.2.tar.xz`. If you unpack that tar ball and check what it contains, you'll find `pbench-local/pbench-move-results-receive/fs-version-00{1,2}` directories that contain (fake) result tar balls. The results in `fs-version-001` are not processed any more with your changes in this PR, but at least some of them should be. My take:

-  `controller-a-bad-link` should just be deleted because it uses the TODO subdir to contain a bad link, but that's 001-specific. 002 reception areas do no have a TODO subdir and do not use links, so we cannot have a bad link.

- `controller-b-with-prefixes` contains tar balls and their MD5 sums plus a TODO subdirectory with links. The whole directory should be moved to the 002 area, but the TODO subdirectory should be deleted.

- `controller-c-missing-md5`: should be moved to the 002 area but the TODO subdir should be deleted.

- `controller-d-duplicate`: should be deleted, since there is an identical setup in the 002 area.

- `controller-e-bad-md5`: should be deleted, since there is an (almost) identical setup in the 002 area.